README.md

```diff
@@ -257,7 +257,7 @@ Offical exam objectives you review and understand in order to pass the test.
 
 #### Other CKS related repos
 
-1. [Stackrox CKS study guide](https://github.com/stackrox/Kubernetes_Security_Specialist_Study_Guide) - Brief and inforative study guide from[Stackrox @mfosterrox](https://www.stackrox.com/authors/mfoster/)
+1. [Stackrox CKS study guide](https://github.com/stackrox/Kubernetes_Security_Specialist_Study_Guide) - Brief and inforative study guide from [Stackrox @mfosterrox](https://www.stackrox.com/authors/mfoster/)
```

Comment:
Small minor spelling error "informative"

Reply:
thank you, fixed, you can squash the commits